utils/tests.py

```
@@ -276,3 +277,40 @@ def test_output_path_parsing_failure(self, mock_subprocess_run):
+
+class TestSearchToolNormalizeName(TestCase):
+    """Unit tests for SearchTool.normalize_name method."""
+
+    def setUp(self):
+        """Set up test fixtures."""
+        self.tool = SearchTool(filename="test")
+
+    def test_normalize_name_removes_read_by(self):
+        """Verify 'read by' and narrator names are removed."""
+        result = self.tool.normalize_name("Great Book read by John Smith")
+        self.assertNotIn("read by", result.lower())
```

Ruff (0.15.1)

[warning] Use a regular `assert` instead of unittest-style `assertNotIn`

Replace `assertNotIn(...)` with `assert ...`

(PT009)